ga4gh/frontend.py

```diff
@@ -372,6 +433,80 @@ def searchVariants(version):
         version, flask.request, app.backend.searchVariants)
 
 
+@app.route('/_login', methods=['GET'])
+def login():
+    """
+    The entry point for getting an authentication token. This sets up an
+    OIDC authorization request and redirects the client to the appropriate
+    URI at the provider.
+
+    :return: Usually a redirect to the authentication provider
+    """
+    if not app.oidcClient:
+        return 'Authentication not enabled.', 200
```

Comment:
Two questions (same below in oidcCallback):

1. Should we return a Flask object here, rather than a tuple?
2. Would it be simpler to just 404 here instead?